build.gradle

```diff
@@ -71,3 +72,7 @@ subprojects {
 
     apply from: '../gradle/publish.gradle'
 }
+
+configure(subprojects.findAll {it.name != 'jaeger-generated'}) {
```

Comment:
I'd prefer to name it more precisely, jaeger-thrift. If or when we have something else that's generated, we can revisit the naming.